Sheikh_Shahzeb/User3_Management_System/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+	def index
+		@users =User.all
+	end
+	def new
+
+	end
+	def create
+		User.create(first_name:params[:first_name],last_name:params[:last_name], email_address:params[:email_address],password:params[:password])
+		redirect_to('/')
+	end
+	def show
+		@user = User.find(params[:id])
+	end
+	def edit
+		@user = User.find(params[:id])
+	end
+	def update
+		User.update(params[:id], first_name:params[:first_name],last_name:params[:last_name], email_address:params[:email_address], password:params[:password])
+		redirect_to '/'
```

Comment:
You might want to create a private method for your params that'll be more secure and you could then use that method to pass the data into both your create and update methods.  For example:
     def user_params
  	params.require(:user).permit(:first_name, :last_name, :email_address, :password)
  end

then in the create or update instead of passing all of that info into the method you would simply pass in user_params